.github/workflows/ci_module.yml

```diff
@@ -344,6 +344,25 @@ jobs:
           OPEN_SEARCH_HTTP_PORT: 9200
           OPENSEARCH_HOSTS: "localhost:9200"
           OPENSEARCH_DSL_AUTOSYNC: False
+      - name: Generate Coverage Report
+        if: always()
+        working-directory: ./openimis/openIMIS
+        run: |
+          python -m coverage xml -o coverage.xml
+          # Only leave actual code part, skip absuolute directory like 
+          # /home/runner/work/openimis-be-invoice_py/openimis-be-invoice_py/current-module/
+          # It's required by circle-ci setup
+          mkdir coverage
+          sed -E 's|/home/runner/work/(.*?)/(.*?)/current-module/|./current-module/|' coverage.xml >> coverage/coverage1.xml
+          sed -E 's|/home/runner/work/(.*?)/(.*?)/current-module/|./|' coverage.xml >> coverage/coverage2.xml
+          
+          cat coverage.xml
+      - name: Coverage results
+        if: success() || failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: coverage.xml
+          path: ./openimis/openIMIS/coverage
```

Comment:
@delcroip this came back because there are subsequent references to `coverage.xml` – unless we have a good reason to remove coverage report I think we should keep it.

Reply:
no ones reads it, everytime I had a look I was skeptical on the result ... Honestly I plan to remove / move other non testing CI because to make the overall CI red for no good reason (It seems that only me worked on the flake8 for instance)

Reply:
I'd suggest keeping it without having it affect CI red/green. If the results are unreliable we should address it instead of throwing the baby out with the bath water.